include/open62541/server_pubsub.h

```diff
@@ -282,6 +282,16 @@ UA_Server_getPubSubConnectionConfig(UA_Server *server,
 UA_StatusCode UA_EXPORT UA_THREADSAFE
 UA_Server_removePubSubConnection(UA_Server *server, const UA_NodeId connection);
 
+UA_StatusCode
```

Comment:
I don't recognize the "enabled" nomenclature.
Does this come from the standard?

The PubSubState that underlies the state machine has five possible states:

```
    UA_PUBSUBSTATE_DISABLED = 0,
    UA_PUBSUBSTATE_PAUSED = 1,
    UA_PUBSUBSTATE_OPERATIONAL = 2,
    UA_PUBSUBSTATE_ERROR = 3,
    UA_PUBSUBSTATE_PREOPERATIONAL = 4,
```
I suggest a single method to interact with the state machine:
```c
UA_Server_setPubSubConnectionState(UA_Server *Server, const UA_NodeId connectionIdent
                  UA_PubSubState state, UA_StatusCode cause);
```
The same for Reader/Writer/ReaderGroup/WriterGroup.
This also very well maps to the internal methods we already have.
Such as 
```c
UA_StatusCode
UA_DataSetWriter_setPubSubState(UA_Server *server,
                                UA_DataSetWriter *dataSetWriter,
                                UA_PubSubState state,
                                UA_StatusCode cause);```

The old public API (like setOperational) can be kept for a little while as a lightweight wrapper.
Before eventually marking it as deprecated.